circle.yml

```diff
@@ -39,11 +39,8 @@ machine:
 
 checkout:
   post:
-    - |
-      git clone --depth 1 ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
-      cd /home/ubuntu/st2/st2-packages
-      git checkout ${CIRCLE_BRANCH} || true
```

Comment:
`RED` is better than building against the wrong branch. I can live with that one.